_maps/toolbox_arenas/bottomless_pit.dmm

```diff
@@ -0,0 +1,409 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/open/misc/asteroid,
+/area/space)
+"b" = (
+/obj/structure/window/reinforced{
+	dir = 8
+	},
+/turf/open/floor/wood,
+/area/space)
+"i" = (
+/obj/structure/flora/bush/grassy/style_3,
+/turf/open/misc/asteroid,
+/area/space)
+"l" = (
+/obj/effect/decal/fakelattice,
+/obj/effect/turf_decal/sand{
+	density = 1
+	},
+/obj/structure/window/reinforced{
+	dir = 4;
+	pixel_y = 2
+	},
+/obj/structure/window/reinforced{
+	dir = 8
+	},
+/obj/structure/window/reinforced{
+	dir = 1
+	},
+/turf/open/floor/pod/light{
+	density = 1
+	},
+/area/space)
+"n" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 8
+	},
+/obj/structure/railing{
+	dir = 8
+	},
+/turf/open/misc/asteroid,
+/area/space)
+"p" = (
+/obj/structure/mecha_wreckage/ripley/mk2,
+/turf/open/misc/asteroid,
+/area/space)
+"q" = (
+/obj/structure/flora/bush/style_4,
+/turf/open/misc/asteroid,
+/area/space)
+"t" = (
+/obj/effect/turf_decal/stripes/line,
+/obj/structure/railing,
+/turf/open/misc/asteroid,
+/area/space)
+"w" = (
+/obj/structure/window/reinforced{
+	dir = 4;
+	pixel_y = 2
+	},
+/turf/open/floor/wood,
+/area/space)
+"z" = (
+/obj/structure/window/reinforced{
+	dir = 8
+	},
+/obj/structure/window/reinforced{
+	dir = 1
+	},
+/turf/open/floor/wood,
+/area/space)
+"A" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/obj/structure/railing{
+	dir = 1
+	},
+/turf/open/misc/asteroid,
+/area/space)
+"C" = (
+/obj/structure/chair,
+/turf/open/floor/wood,
+/area/space)
+"D" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/obj/structure/railing{
+	dir = 1
+	},
+/obj/structure/sign/barsign{
+	name = "bottomless pit";
+	icon_state = "bottomlesspit1";
+	icon = 'modular_event/tournament/sign2.dmi'
+	},
```

Comment:
this will need to be defined in it's own .dm file as a new subtype in order for it to work unfortunately

for example `/obj/structure/sign/barsign/bottomless` in modular_event/arena_fluff/bottomless_sign.dm